app/sdil/config/RoutesWiring.scala

```diff
@@ -28,10 +30,10 @@ import sdil.controllers.{VariationsController, RegistrationController => Uniform
 import uk.gov.hmrc.auth.core.AuthConnector
 import uk.gov.hmrc.http.cache.client.{SessionCache, ShortLivedHttpCaching}
 import uk.gov.hmrc.play.bootstrap.http.{FrontendErrorHandler, HttpClient}
+import uk.gov.hmrc.play.health.HealthController
 
 trait RoutesWiring extends CommonWiring {
   val errorHandler: FrontendErrorHandler
-  val applicationLifecycle: DefaultApplicationLifecycle
```

Comment:
I suspect this is not used or is part of another package?

Reply:
I expect this to be available on application create. like you can see in fakeApplicationSpec. and since the application runs seems to be fine, just waiting for the acceptance tests to double check these type of changes.